tests/pr_test.go

```diff
@@ -100,6 +100,7 @@ func TestRunFSCloudExample(t *testing.T) {
 	options := setupOptions(t, "cos-fscloud", fsCloudTerraformDir)
 	options.TerraformVars["bucket_existing_hpcs_instance_guid"] = permanentResources["hpcs_south"]
 	options.TerraformVars["bucket_hpcs_key_crn"] = permanentResources["hpcs_south_root_key_crn"]
+	options.TerraformVars["management_endpoint_type_for_bucket"] = "public"
```

Comment:
as per https://github.com/terraform-ibm-modules/terraform-ibm-cos/pull/480/files#r1433917291 I think we default the variable to public